scipy/stats/_multivariate.py

```diff
@@ -1421,8 +1421,9 @@ def mean(self, alpha):
 
         Returns
         -------
-        mu : scalar
-            Mean of the Dirichlet distribution
+        mu : ndarray
```

Comment:
Realize now that that was the original bug report - rather than changing the doc I think that ``var`` should be made consistent with the other methods. It's a very minor backwards compat break, but should be acceptable (needs a mention in the 1.0 release notes) imho.